pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -209,3 +211,15 @@ func (c *contract) callTx(ctx context.Context, callData []byte) ([]byte, error)
 	}
 	return result, nil
 }
+
+func (c *contract) setFee(ctx context.Context, txHash common.Hash) {
+	fee, err := c.txService.TransactionFee(ctx, txHash)
+	if err != nil {
+		c.logger.Info("transaction fee error:", err)
+	}
+	c.accumulativeFee = fee
```

Comment:
this needs to be similar to the total reward where we increment the accumulativeFee eg `accumulativeFee += trx_fee`

Reply:
This lower layer should only return the fee. The business logic of adding fees should be done in node status, which I am already doing.

https://github.com/ethersphere/bee/blob/a670af9fb998e88c57dfac43ad7e6b6f01ddfd78/pkg/storageincentives/nodestatus.go#L110

Reply:
I propose that we can simply return the trx fee directly from the call so 
`a.contract.Reveal`, `a.contract.Commit`, and `a.contract.Claim` would also return the associated fees that you can increment in `nodeState`

Reply:
This is fixed.